src/mpi/romio/adio/ad_quobytefs/Makefile.mk

```diff
@@ -0,0 +1,26 @@
+## -*- Mode: Makefile; -*-
+## vim: set ft=automake :
+##
+## Copyright 2019 Quobyte Inc. All rights reserved.
+## Author: alexey@quobyte.com
+##     See COPYRIGHT in top-level directory.
```

Comment:
Contribution information goes into the top-level copyright file.  Institute-specific contribution information are maintained in the PRs, not in individual files.

Reply:
> > > @reflectored This should be rebased on to master.
> > 
> > 
> > @pavanbalaji There is another problem, I'm missing in contributor list:
> > Checking PR author [reflectored]
> > checking in team pmodels-core ...
> > checking in team mpich-collab ...
> > Author not found.
> 
> Yes, we'll need either an individual or an institutional CLA signed for that: http://www.mpich.org/documentation/contributor-docs/

We have sent it to Robert Latham. Please provide me with your email address and I will forward you the contract.

Reply:
Sorry, @roblatham00 did add it to the CLA folder.  But looks like out GitHub check didn't get updated.  @hzhou can you do that?

Reply:
Please note that my above comment was unrelated to the CLA.  It's about where the copyright information should go.

Reply:
> Sorry, @roblatham00 did add it to the CLA folder. But looks like our GitHub check didn't get updated. @hzhou can you do that?

Sure. Just need add author's github userid into `mpich-collab` team. Excuse my ignorance, does the author belong to any institute or representing individually?

Reply:
Found it. Should I add a `mpich-quobyte` team or aggregate into `mpich-other`?

Reply:
I think `mpich-other` is fine for now.